GETTING_STARTED.md

```diff
@@ -0,0 +1,380 @@
+ * [1. Preface](#1-preface)
+    * [1.1. Information is widely spread across the company](#11-information-is-widely-spread-across-the-company)
+    * [1.2. Connecting the dots](#12-connecting-the-dots)
+    * [1.3. Editing / Publishing Separation](#13-editing--publishing-separation)
+ * [2. Getting started](#2-getting-started)
+    * [2.1. Data retrieval](#21-data-retrieval)
+    * [2.2. Data types &amp; column kinds](#22-data-types--column-kinds)
+       * [Primitive data types](#primitive-data-types)
+          * [Examples](#examples)
+       * [Higher data types](#higher-data-types)
+          * [<code>multi-language</code> and <code>multi-country</code>](#multi-language-and-multi-country)
+          * [<code>link</code>](#link)
+          * [<code>concat</code>, and <code>group</code>](#concat-and-group)
+          * [<code>attachment</code>](#attachment)
+
+
+# 1. Preface
+
+GRUD is an acronym and stands for "generic relational enterprise database". The following document should give a short introduction to GRUD and its principles.
+
+## 1.1. Information is widely spread across the company
+In today's companies data and information is widely spread across divisions and departments. We saw the need for storing Excel-like data in a structured and centralized way. A company could and should take advantage of connecting all this widespread data.
```

Comment:
It might make sense to add a story to this, like: We have seen data in a lot of places. Some information lives in plain text files, some in Excel-Sheets, you find some in PDFs or even just on a website. All these data-silos can be replaced one by one, by adding their additional information to the GRUD. This way it is not necessary to duplicate content and maintain information multiple times. The result is a centralised database with all data connected. The GRUD structure can grow and scale depending on the data necessary for the various channels.

Reply:
Good idea. Maybe not only suggest moving data to the GRUD, but also explicitly point out that the content for channels which have been manually maintained before can then be automatically generated and enhanced from that centered data base.

I know that your text implies that, but it's one of the stronges points for GRUD, and thus I'd explicitly write it, as it will also require content maintainers to change the way they think and work.

Reply:
Good points but I don't want to write text for a marketing brochure. This is a document which you give a short introduction to GRUD. More from a technical point of view. Nevertheless I changed the text a little bit to reflect your comments.